app/networkapi/people/views.py

```diff
@@ -1,3 +1,28 @@
 from django.shortcuts import render
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+from networkapi.people.models import Person
```

Comment:
generally try to follow the convention of having a blank line between core deps and app deps